### PR TITLE
Updated PHP coding standards

### PR DESCRIPTION
**Pull request by mit456**

### CODE REVIEW

File: php.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+  }
+
+  private function printFoo()
+  {
+    //Variables
+    $foo = "Foo";
+
+    /function calls
+    foo = $this->updateFoo($foo);
+  }
```
Comment (rampr) [inline php.md]: need newline here

File: php.md [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+  $a = 1;
+
+  function2();
+  function3($a);
+}
+
+//bad
+class Foo()
+{
+  //Constructor decleration
```
Comment (rampr) [inline php.md]: declaration(typo)

File: php.md [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+  function Foo()
+  {
+    //Do stuffs
+  }
+}
+//Good
+class Foo()
+{
+  //Constructor decleration
```
Comment (rampr) [inline php.md]: same typo

File: php.md [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+//Bad
+$a='Foo';
+//Good
+$a = 'Foo';
+
+//if-else statement
+
+//Bad
+if($a > $b)
+  //execute single line
```
Comment (rampr) [inline php.md]: use new lines, will make it more readable

File: php.md [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+//Bad
+$VariableNotLikeThis = 0;
+$Variable_Not_Like_This = 0;
+
+//Good
+$variable_like_this = 0;
+```
+## Spacing and empty lines
+
+* Give necessary empty lines to make code more readble
```
Comment (rampr) [inline php.md]: readable(typo)

**Comment by jishnu7** [comment]
LGTM

File: php.md (hunk 4 above)
Comment (mit456) [inline php.md]: Fixed

**Pull request merged**